nav2_simple_navigator/include/nav2_simple_navigator/simple_navigator.hpp

```diff
@@ -34,7 +34,7 @@ class SimpleNavigator : public rclcpp::Node
 
   nav2_tasks::TaskStatus navigateToPose(const nav2_tasks::NavigateToPoseCommand::SharedPtr command);
 
-protected:
+private:
```

Comment:
Why the switch to private? If anybody ever felt the need to inherit from SimpleNavigator, it seems they'd likely need access to these variables.